src/main.rs

```diff
@@ -185,6 +188,16 @@ fn settings() -> io::Result<Settings> {
                 .number_of_values(1)
                 .help("Prevents deletion of images for which repository:tag matches <REGEX>"),
         )
+        .arg(
+            Arg::with_name(DELETION_CHUNK_SIZE_OPTION)
+                .value_name("DELETION CHUNK SIZE")
+                .short("d")
+                .long(DELETION_CHUNK_SIZE_OPTION)
```

Comment:
I noticed the `--keep` option (right above this one) has `.number_of_values(1)`. Should we add that here too?